### ISSUE TITLE
Few minor formatting issues

### ISSUE THREAD

**Issue by vr8hub** [maxwellaires/l-frank-baum_the-marvelous-land-of-oz#2]
1. The business card in chapter 12 should be wrapped in a blockquote.
2. The sign in chapter 12 doesn't need a class; you can just use `<b>` on the text.
3. Ditto for the first line of the pill instructions in chapter 18.
4. The `<strong>`'s in those instructions and in the dedication should just be plain `<b>`. `<strong>` is reserved for something like shouting; `<b>` is used when we just need caps for things like these two situations.
5. In the case of the dedication, I think putting "the author" on a separate line and right-justifying it will make it read a little better without a period. And it should be titlecased like the scans (The Author).

**Comment by maxwellaires** [comment]
What exactly are you referring to for 2 -- I don't think there's a sign with a class in chapter 12. Do you mean the one in chapter 5?

**Comment by vr8hub** [comment]
Yes, so sorry, I must have copied the chapter from 1) when I was typing. I'm referring to the one in chapter 5.

**Comment by vr8hub** [comment]
Sorry, I just noticed the first line of the Nikidik's instructions in chapter 18 aren't small-capped (`<b>`).
